pkg/apply/utils.go

```diff
@@ -109,3 +109,24 @@ func getHostArch(sshClient ssh.Interface) func(string) v2.Arch {
 func GetHostArch(sshClient ssh.Interface, ip string) string {
 	return string(getHostArch(sshClient)(ip))
 }
+
+func GetImagesDiff(current, desired []string) []string {
```

Comment:
https://github.com/labring/sealos/blob/a02ebcafefbeb31b284ee8786987d7f7c6455721/pkg/utils/strings/strings.go#L174

can use this method to get different images

Reply:
do I need to consider the uniqueness of the image for this function as it doesn't seem to have been taken into account?